server/controllers/documents.js

```diff
@@ -0,0 +1,204 @@
+import models from '../models';
+import jwt from 'jsonwebtoken';
+import config from '../config/config';
+
+const Documents = models.Documents;
+const Users = models.Users;
+
+const checkDocOwner = (docId, queryId) => {
+  if (parseInt(docId, 10) === parseInt(queryId, 10)) return true;
+};
+
+const checkUserRole = (roleId) => {
```

Comment:
Use a more descriptive name. I looks to me like this function would be better called `isWriter()` or `checkIfWriter()`